src/index.js

```diff
@@ -1,17 +1,9 @@
 import React from 'react';
-import ReactDOM from 'react-dom/client';
+import ReactDOM from 'react-dom';
+import App from './App.js'; // Add .js extension here
 import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+// Rest of the code...
 
-const root = ReactDOM.createRoot(document.getElementById('root'));
-root.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>
-);
+const root = document.getElementById('root');
 
-// If you want to start measuring performance in your app, pass a function
-// to log results (for example: reportWebVitals(console.log))
-// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
-reportWebVitals();
+ReactDOM.createRoot(root).render(<App />);
```

Comment:
- [ ] Kindly make sure you app componet is wrapped in the <Provider> so that your  redux store will be well linked in you project

```.js
import { Provider } from 'react-redux';
import store from './store/index.js';
// at the top


// wrap the app component in provider from redux-react
    <Provider store={store}>
      <App />
    </Provider>
);

![image](https://github.com/david30-maker/hello-world-react-two/assets/65068771/f8ec4a51-a54c-4c1d-abfe-ad5dc608dae4)